.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,72 @@
+# This workflow will ensure all PRs are labeled so we can later create releases
+
+name: Prerelease
+
+on: push
+
+jobs:
+  prerelease:
+    if: "contains(github.event.head_commit.message, '[prerelease]')"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: ${{ github.head_ref }}
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 16
+          registry-url: https://registry.npmjs.org/
+      - name: Install dependencies
+        run: npm ci
+      - name: Set up git credentials
+        run: |
+          git config --global user.name 'ci'
+          git config --global user.email 'deploy@monday.com'
+      - name: "Wait for tests to succeed"
+        uses: fountainhead/action-wait-for-check@v1.1.0
+        if: false
+        id: wait-for-tests
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          checkName: "test"
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+          timeoutSeconds: 1200
+          intervalSeconds: 20
+      - name: "Tests failed - exit"
+        if: false && steps.wait-for-tests.outputs.conclusion != 'success'
```

Comment:
should this be `false` or this is the tests?

Reply:
for tests, will be removed

Reply:
thanks